packages/ember-routing-htmlbars/tests/helpers/render_test.js

```diff
@@ -31,53 +21,13 @@ import { renderHelper } from "ember-routing-htmlbars/helpers/render";
 import { actionHelper } from "ember-routing-htmlbars/helpers/action";
 import { outletHelper } from "ember-routing-htmlbars/helpers/outlet";
 
-function appendView(view) {
-  run(function() { view.appendTo('#qunit-fixture'); });
-}
-
-function set(object, key, value) {
-  run(function() { emberSet(object, key, value); });
-}
```

Comment:
@rwjblue the confusion around `set` being run inside and outside of the `run` loop stems from this line. That's why I've renamed to `runSet` to make the distinction explicit.